packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -2754,19 +2749,8 @@ function commitMutationEffectsOnFiber(
             const oldProps =
               current !== null ? current.memoizedProps : newProps;
             const type = finishedWork.type;
-            // TODO: Type the updateQueue to be specific to host components.
-            const updatePayload: null | UpdatePayload =
-              (finishedWork.updateQueue: any);
-            finishedWork.updateQueue = null;
```

Comment:
Confirmed by logging in `yarn test` that `  finishedWork.updateQueu` is always `null` here